### DISCUSSION TITLE
Yggtorrent

### DISCUSSION THREAD

**Discussion by scott451** [pymedusa/Medusa#11337] [General]
Hello,

I have an on-again off-again relationship with Medusa and Ygg. It seems to start and stop working on its own volition. I have the latest version 1.0.16 installed but I get no results from the internal Ygg or from a Ygg that I have configured with Prowlarr (and use successfully with Radarr).

I run Medusa, as well as Radarr and Prowlarr, on a Synology RS-1221 running DSM 6.

Ii know that this is not the fault of Medusa. It's probably a mis-configuration on my part somewhere. Or maybe Ygg has changed their address again and I just need to be patient. Thnanks for any insight.   :-)

Scott

**Comment by Mika3578** [comment]
It appears that you might need to install FlareSolverr and configure it within Prowlarr. This could help to resolve the issues you're facing with Ygg not returning any results. FlareSolverr is designed to help bypass Cloudflare's anti-bot measures, which could potentially be causing the issues you're experiencing. Additionally, it's important to ensure that the categories for the tracker are correctly configured within Medusa. This could also affect the results you're getting.
